### ISSUE TITLE
Add jobs for new repo jbpm-work-items

### ISSUE THREAD

**Issue by tsurdilo** [kiegroup/kie-jenkins-scripts#93]
We need to create jobs for https://github.com/kiegroup/jbpm-work-items please.

**Comment by tsurdilo** [comment]
Branches: just master branch
PR jobs : yes
Repository-list: yes

**Comment by mswiderski** [comment]
would be good to take a look at the versioning we have in this repo and how shall we deal with them. Here are few ideas we were looking into:

- use the same version as KIE projects - makes it easy to upgrade and makes it easy to users to know what versions are compatible with wich version of jBPM - plus it will simplify release of the work items - we release all with updated version every time jBPM is released, regardless if there were any changes to the work items or not
- use single version for all modules in jbpm-work-items repository - like we have now 1.0.0.Final - it should be then upgraded jbpm is released. Problem here is how to increase version - shall it be always major version change, minor version change or? Regardless if there are modules that have changes in between jBPM releases they will only be released once with jBPM. this might be bit confusing to know which version of the work item is compatible with what version of jBPM since they will have different versions, could be most likely solved by documentation
- use independent versions for each work item and only release them when they change. Whenever jBPM is released we release work items with their latest version, similar as in point two - bit confusing what version of the work item is compatible with what version of jBPM

any feedback / recommendation more than welcome
@mbiarnes @psiroky

**Comment by psiroky** [comment]
+1 to use KIE version

**Comment by tsurdilo** [comment]
@mswiderski @psiroky thanks! updated the jbpm-workitems module poms to use the kie version.

**Comment by tsurdilo** [comment]
any news on this ? any eta on when we could add the scripts?

**Comment by mswiderski** [comment]
@mbiarnes @psiroky guys, can someone help out with this? We would like to have this done (new service repository, PR based builds and upload to service repo at release time) done for 7.5 community

**Comment by mbiarnes** [comment]
@mswiderski @tsurdilo IIUC the jbpm-work-items will not be now in repository-list.txt but will have the KIE-version, so when releasing we have to update the versions. Since this doesn't go into repository-list.txt this has to be done manually.  
Will add his to PR jobs today.

**Comment by mswiderski** [comment]
@mbiarnes we will add this to repository-list.txt once we have all working as expected so it should be done before the release. But for now we don't want to change it until we have everything covered.

**Comment by mbiarnes** [comment]
@mswiderski OK - so right now - I mean until this was not added tor repository-list.txt it is not needed for the daily builds or the releases?

**Comment by mswiderski** [comment]
it's not needed, as long as we can trigger it on PR and manually then we are good

**Comment by psiroky** [comment]
Looking at https://github.com/kiegroup/jbpm-work-items -- the POMs and dependencies there will need a major clean-up, before we can add the repository to `repository-list.txt`. And in case workitems should be part of a product releases, then we will need even more rigorous review of all the 3rd party deps.

**Comment by mswiderski** [comment]
@psiroky these are not meant to be released with product, that's why they are in separate repo and each of them is marked as contribution. Purely for comment consumption and hopefully community contribution as well.

@tsurdilo could you please reach out to @psiroky regarding clean up

**Comment by tsurdilo** [comment]
@psiroky ok sure i will clean up anything you suggest. could you tell me what should change? Thanks.

**Comment by tsurdilo** [comment]
@mswiderski @psiroky : https://github.com/kiegroup/jbpm-work-items/pull/12 
please take a look at let me know if there is anything else you would like for me to change to we can include this in repository-list. Thanks!

**Comment by psiroky** [comment]
This should be done now. The deploy job is not working, because no `distributionManagement` is currently configured. That should get fixed once kiegroup/jbpm-work-items#12 is merged.

I am closing this. Please re-open in case you find some issue.
